src/index.js

```diff
@@ -72,9 +72,9 @@ class ESLintWebpackPlugin {
     const options = {
       ...this.options,
       exclude: parseFiles(
-        this.options.exclude || 'node_modules',
+        this.options.exclude || [],
         this.getContext(compiler)
-      ),
+      ).concat('**/node_modules/**'),
```

Comment:
I don't think it is good idea, sometimes developers need linting strange things, so if you provide own `exclude` you can setup any values, i.e. and enable linting `node_modules`